test/CMakeLists.txt

```diff
@@ -1745,7 +1746,8 @@ ecbuild_add_test( TARGET  test_${PROJECT_NAME}_smap9km_ssm
                             "${Python3_EXECUTABLE} ${CMAKE_BINARY_DIR}/bin/smap9km_ssm2ioda.py
                             -i testinput/SMAP_L2_SM_P_E_36365_D_20211122T013945_R18240_001.h5
                             -o testrun/smap9km_ssm.nc
-                            -m maskout"
+                            -m maskout
```

Comment:
really think passing the string `maskout` is a bit odd and yes can change here but would recommend adding just the `-m` argument which changes a boolean to true and removing the depth entirely and have the default be in the python converter

it is potentially a bit hidden but do not want people thinking they can change the retrieval somehow

Reply:
@BenjaminRuston I'm also OK with hard coding the SMAP depth, since this is a fixed characteristic of L-band. The only reason I can think of that the observed depth would change would be if we changed the interpolation algorithms to no longer act on the midpoint of each layer - but this would be a much larger code change anyway. We might also want to trick the interpolation into using only model layer 1 in h(x), even if layer 1 is not 5 cm, by setting the observed depth to whatever the depth of layer 1 is, but this is an ugly work-around, so I think being forced to change the default value in this case is appropriate.

Reply:
@BenjaminRuston, I would rather than keep this flexibility for the user to avoid often modify code. We have a default option, if you select -m default. It is more flexible for the depth is not exact the same 0.025m, as Clara said for interpolation or something else. We can update README.md to tell the user how to use it. This should totally fine, I think.

Reply:
This is my opinion. If we want to update, certainly user friendly update is preferred.

Reply:
I do not think CMake is user friendly and a ctest is not what people should be modifying unless they are changing the file.

the converter should have the flexibility... I guess I will talk with Clara about this and we can make the call

Reply:
please change and remove the string argument `maskout` and make it a boolean

Reply:
if you like I can push changes up directly to your branch